.github/workflows/required.yml

```diff
@@ -0,0 +1,91 @@
+name: Required Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Just curious, does this mean when we create a branch we need to adjust these branch names? Or is it possible to get the current branch name from a variable?

Reply:
With current code we need to adjust the branch names. There is no variable as such to get the branch name.

Removing the following code and adding `on: [push, pull_request]` triggers CI for push/PR against all branches

````
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
````